src-v2/NTS.Domain.Core/Aggregates/Participations/NotQualified.cs

```diff
@@ -66,8 +71,10 @@ public FailedToQualify(params FTQCodes[] codes)
             throw new DomainException($"'Failed to Complete' requires a writen explanation from officials. Please provide 'complement'");
         }
         Codes = codes;
+        EliminationCode = FAILED_TO_QUALIFY;
```

Comment:
why don't you pass it to the base ctor instead?

Reply:
Because for correct deserialization the ctor needs to be decorated with the [JsonConstructor] attribute. However, it is already used once in this class. Another usage creates ambiguity and results in runtime errors.

Reply:
"results in runtime errors" I presume you meant "compile-time errors"? 

Using another `JsonConstructor` attribute is a nonsense, because that attribute is used to select a ctor that used during deserialization to instantiate the object. If there are two attributes how is the deserializer to know which one to select? 

That said this has nothing to do with base ctor. If this ctor isn't used during deserialization, then no changes to it will affect deserialization. For example if you set `EliminationCode`  to `Retired`, but the the serialized data does not have `Retired` value, your object after deserialization will contain whatever the value is in the serialized data. It's not going to be `Retired`, because that's not the ctor that the deserializer used. All of this is a long way of saying, you seem to be confusing constructors.

In fact, as is now, `NotQualified` has empty protected ctor. This then allows children to be instantiated without `EliminationCode` which could be a problem.